pkg/integration/replication/suite.go

```diff
@@ -305,6 +308,20 @@ func (suite *baseReplicationTestSuite) SetupCluster(syncReplicas, syncAcks, asyn
 	}
 
 	wg.Wait()
+
+	uuids := make(map[string]struct{}, 1+suite.GetFollowersCount())
+
+	uuids[suite.master.UUID(suite.T()).String()] = struct{}{}
+
+	for _, f := range suite.followers {
+		uuid := f.UUID(suite.T()).String()
```

Comment:
This disables ability to run tests with docker runner - every cluster setup will panic - maybe we should not be so strict here?